tools/eslint-rules/prefer-primordials.js

```diff
@@ -57,8 +57,18 @@ function getDestructuringAssignmentParent(scope, node) {
   return declaration.defs[0].node.init;
 }
 
-const identifierSelector =
-  '[type!=VariableDeclarator][type!=MemberExpression]>Identifier';
+const parentSelectors = [
+  // We want to select identifier that refer to another reference, not the one
```

Comment:
```suggestion
  // We want to select identifiers that refer to other references, not the ones
```

Alternatively, change "create" on the next line to "creates".